docs/get-started/wpf/datatemplates.md

```diff
@@ -22,7 +22,7 @@ For example, this code adds a data template to display the view model class `MyV
     <!-- Assuming that DataContext.Foo is an object of type
          MyApp.ViewModels.FooViewModel then a red border with a corner
          radius of 8 containing a TextBox will be displayed here -->
-    <ContentControl Content="{Binding Foo}"/>
+    <ContentControl DataType="viewmodels:UserControl1ViewModel" Content="{Binding Foo}"/>
```

Comment:
UserControl1ViewModel is not mentioned anywhere on this page, and is a bit confusing.
Also, it would make more sense to add DataType on the top-level element here - UserControl.